recOrder/cli/utils.py

```diff
@@ -37,31 +39,44 @@ def create_empty_hcs_zarr(
                 dtype=dtype,
                 transform=[TransformationMeta(type="scale", scale=scale)],
             )
+        else:
+            position = output_plate[position_key_string]
+
+        # Check if channel_names are already in the store, if not append them
+        for channel_name in channel_names:
```

Comment:
Please document the new channel appending behavior.

Reply:
Thanks! Documentation improved.